### PR TITLE
(RE-6464) Remove Custom Actions/Sequences from Generic Product file

### PR DESCRIPTION
**Pull request by jcoconnor**
Custom Actions and sequences are to be put into the puppet-agent or
other product specific code areas.

This PR removes the Custom action sequences from the product file
completely. A PA side PR will be used to create the custom actions
there.

### CODE REVIEW

**Comment by stahnma** [comment]
:+1: from me, but I have minimal context.

File: resources/windows/wix/project.wxs.erb [hunk 1]
```diff
@@ -41,33 +41,5 @@
       </Component>
     </Directory>
 
-    <SetProperty Id="ARPINSTALLLOCATION" Value="[INSTALLDIR]" After="CostFinalize" />
-    <Property Id="INSTALLDIR">
```
Comment (McdonaldSeanp) [inline resources/windows/wix/project.wxs.erb]: Are these properties part of the custom actions?

Comment (jcoconnor) [inline resources/windows/wix/project.wxs.erb]: Yes - will move them now.

Comment (McdonaldSeanp) [inline resources/windows/wix/project.wxs.erb]: Okay, sounds good

**Comment by McdonaldSeanp** [comment]
Does WiX complain when this specific PR is run through, specifically can you get in to the actual light linker step?

**Comment by jcoconnor** [comment]
@McdonaldSeanp This is actually a minor change and Wix links fine with it.

**Comment by McdonaldSeanp** [comment]
Great

**Pull request merged**